pkg/kubelet/kubelet_node_status.go

```diff
@@ -117,6 +118,7 @@ func (kl *Kubelet) tryRegisterWithAPIServer(node *v1.Node) bool {
 	requiresUpdate := kl.reconcileCMADAnnotationWithExistingNode(node, existingNode)
 	requiresUpdate = kl.updateDefaultLabels(node, existingNode) || requiresUpdate
 	requiresUpdate = kl.reconcileExtendedResource(node, existingNode) || requiresUpdate
+	requiresUpdate = kl.reconcileHugePageResource(node, existingNode) || requiresUpdate
```

Comment:
I need to catch-up on latest state of this code, but we traditionally do not support hot-swap of other resources like cpu or memory for a same named node without a restart.  i thought i had seen a similar pr with an update in this space last year, and so I need to refresh my own memory if we handled hugepages separately.

Reply:
well, devices would get updated as a counted resource.

Reply:
@odinuge ok, this makes sense.